hvplot/converter.py

```diff
@@ -654,17 +659,6 @@ def __init__(
         self.tiles_opts = tiles_opts or {}
         self.sort_date = sort_date
 
-        # Import geoviews if geo-features requested
-        if self.geo or self.datatype == 'geopandas':
-            try:
-                import geoviews  # noqa
-            except ImportError:
-                raise ImportError(
-                    'In order to use geo-related features '
-                    'the geoviews library must be available. '
-                    'It can be installed with:\n  conda '
-                    'install geoviews'
```

Comment:
Refactored it into geoviews is available, and moved it up because `self.crs = ...` was crashing without geoviews.